django/contrib/staticfiles/storage.py

```diff
@@ -86,7 +86,7 @@ class HashedFilesMixin:
                 ),
                 (
                     (
-                        r"(?m)(?P<matched>)^(/\*#[ \t]"
+                        r"(?m)^(?P<matched>/\*#[ \t]"
```

Comment:
This change is untested. Add an example with data URI to the `source_map.css`.

Reply:
Done